src/Widgets/Shapes/ShapeCircle.vala

```diff
@@ -0,0 +1,66 @@
+/*
+* Copyright (c) 2017
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public
+* License as published by the Free Software Foundation; either
+* version 2 of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* General Public License for more details.
+*
+* You should have received a copy of the GNU General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+* Boston, MA 02110-1301 USA
+*
+* Authored by: Alessandro Castellani <castellani.ale@gmail.com>
+*/
+
+/**
+ * ShapeCircle is a child class of the parent CanvasItem {@link Clutter.Actor} that handles move, rotate, scale, etc.
+ *
+ * This is a specific shape class to handle the generation of a Circle geometry
+ */
+public class GtkCanvas.ShapeCircle : GtkCanvas.CanvasItem {
+    public ShapeCircle (string color, double rotation) {
+        this.color = color;
+        this.rotation = rotation;
+
+        real_x = 0;
+        real_y = 0;
+
+        real_w = 100;
+        real_h = 100;
```

Comment:
Instead of using these values, use https://github.com/Philip-Scott/libgtkcanvas/pull/13/files#diff-5f880ccfd30c8b8d210c6ab7dc02f8c3L109

In the future, the real_n properties will have it's own get/set methods, but if you're going to set all four use the function insteadd to prevent extra calls :)